docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -184,28 +193,28 @@ is not enabled can be configured using configuration properties like
 ``dynamic-filtering.large-partitioned.range-row-limit-per-driver`` and their
 equivalent for broadcast join distribution type.
 
-The properties based on ``max-distinct-values-per-driver`` and ``max-size-per-driver``
-define thresholds for the size up to which dynamic filters are collected in a
-distinct values data structure. When the build side exceeds these thresholds,
-Trino switches to collecting min and max values per column to reduce overhead.
-This min-max filter has much lower granularity than the distinct values filter.
-However, it may still be beneficial in filtering some data from the probe side,
-especially when a range of values is selected from the build side of the join.
-The limits for min-max filters collection are defined by the properties
-based on ``range-row-limit-per-driver``.
+The properties based on ``max-distinct-values-per-driver`` and
+``max-size-per-driver`` define thresholds for the size up to which dynamic
+filters are collected in a distinct values data structure. When the build side
+exceeds these thresholds, Trino switches to collecting min and max values per
+column to reduce overhead. This min-max filter has much lower granularity than
+the distinct values filter. However, it may still be beneficial in filtering
+some data from the probe side, especially when a range of values is selected
+from the build side of the join. The limits for min-max filters collection are
+defined by the properties based on ``range-row-limit-per-driver``.
 
 Dimension tables layout
 -----------------------
 
 Dynamic filtering works best for dimension tables where
 table keys are correlated with columns.
 
-For example, a date dimension key column should be correlated with a date column,
-so the table keys monotonically increase with date values.
-An address dimension key can be composed of other columns such as
+For example, a date dimension key column should be correlated with a date
+column, so the table keys monotonically increase with date values. An address
```

Comment:
If so inclined, one _could_ eliminate the nefarious "should be" with something like:
For example, correlate a date dimension key column with a date column
so that the table keys monotonically increase with date values. ...